2996_reflection/reflection.md

```diff
@@ -3919,7 +3908,7 @@ Add new paragraphs before the last paragraph of [basic.fundamental]{.sref} as fo
 
 A reflection is said to _represent_ the corresponding construct.
 
-[A reflection of a value can be produced by library functions such as `std::meta::constant_of` and `std::meta::reflect_constant`]{.note}
+[A reflection of a value can be produced by library functions such as `std::meta::constant_of`, `std::meta::template_arguments_of`, and `std::meta::reflect_constant`.]{.note}
```

Comment:
Eh, I'd revert this. I don't want this to have to be an exhaustive list.

Reply:
Done